### ISSUE TITLE
cmake for cthyb and dft_tools

### ISSUE THREAD

**Issue by pdelamora** [TRIQS/triqs#232]
Few months ago I installed TRIQS and cthyb in
~/TRIQS
in there I put
~/TRIQS/src
~/TRIQS/build
~/TRIQS/src-cthyb
~/TRIQS/build-cthyb
with the corresponding files and libraries

Now I wanted to install dft_tools into 
~/TRIQS/src-tools
~/TRIQS/build-tools

so I typed
git clone https://github.com/TRIQS/dft_tools.git src-tools
and in
~/TRIQS/build-tools
I typed
cmake ../src-tools

and I got
.......................
CMake Error at CMakeLists.txt:15 (find_package):
  By not providing "FindTRIQS.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "TRIQS", but
  CMake did not find one.

  Could not find a package configuration file provided by "TRIQS" with any of
  the following names:

    TRIQSConfig.cmake
    triqs-config.cmake

  Add the installation prefix of "TRIQS" to CMAKE_PREFIX_PATH or set
  "TRIQS_DIR" to a directory containing one of the above files.  If "TRIQS"
  provides a separate development package or SDK, be sure it has been
  installed.
.......................
But the awkward thing is that I reloaded from the start into a new directory
TRIQS-prueba
 and for
TRIQS
everything OK

cthyb and dft_tools
I got the same problem even though I had installed cthyb some months ago without a problem

I tried may things such as;
cmake -DTRIQS_PATH=/home/pablo/TRIQS  ../src-tools
cmake -DTRIQS_PATH=~/TRIQS ../src-tools
cmake -DCMAKE_PREFIX_PATH=../build/Config/TRIQSConfig.cmake ../src-tools
This last one since in 
../build/Config/TRIQSConfig.cmake
there is 
TRIQSConfig.cmake

.....................
Summing up;
I could install
TRIQS
but for 
dft_tools
cthyb
I had problems in spite that I had installed cthyb before without problems
.......................

Thanks
Pablo

**Comment by lmessio** [comment]
in ~/TRIQS/build-tools
try
cmake ../src-tools -DTRIQS_PATH=$~/TRIQS/build/INSTALL_DIR

**Comment by pdelamora** [comment]
Thanks Imessio
I tried your
*** cmake ../src-tools -DTRIQS_PATH=$~/TRIQS/build/INSTALL_DIR
and I got exactly the same as when I typed
*** cmake ../src-tools
(see above)

**Comment by parcollet** [comment]
TRIQS_PATH has to be absolute.
